### PR TITLE
Update gitignore from Github .gitignore template

### PR DESCRIPTION
**Pull request by nkbt**
Fixes especially annoying npm-debug.log

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,21 @@
...
+*.pid
+*.seed
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# node-waf configuration
+.lock-wscript
+
+# Dependency directory
```
Comment (chenglou) [inline .gitignore]: lol, I think we're fine omitting this comment

Comment (nkbt) [inline .gitignore]: hehe, just one from Github template... Will clean it up

**Comment by chenglou** [comment]
Cool, next step is for me to get a few API tweaks in, then we can start putting stuff in `src/` and update all the lint/test/webpack paths if there's any,

**Comment by chenglou** [comment]
^ heads up to @dariocravero and @iclanzan who's decoupling the stepper function. Sorry for moving things around so fast. You folks got on the bandwagon early =)

**Comment by nkbt** [comment]
Yep. Awesome. I was thinking of fixing lint/syntax issues before moving files around. But order does not matter.

**Comment by chenglou** [comment]
Yeah moving will come later. I probably won't be done with the API today

**Pull request merged**